crates/boundless-market/build.rs

```diff
@@ -222,11 +236,44 @@ fn copy_artifacts() {
                 .unwrap()
                 .trim_start_matches("0x");
 
-            // Write to new file with .hex extension
-            let dest_file = dest_path.join(format!("{contract}.hex"));
```

Comment:
When https://github.com/alloy-rs/core/issues/659 gets resolved we could keep the .hex approach

Reply:
yeah we might want to revert using hex files once that gets merged into a release